nusight2/src/client/components/checkbox/view.tsx

```diff
@@ -13,7 +12,7 @@ export interface CheckboxProps {
   onChange(event: ChangeEvent<HTMLInputElement>): void
 }
 
-export const Checkbox: StatelessComponent<CheckboxProps> = (props: CheckboxProps) => {
```

Comment:
`StatelessComponent` is now `PureComponent`. We don't need the type here though.